discord/poll.py

```diff
@@ -0,0 +1,641 @@
+
+        if self.emoji is not None:
+            if isinstance(self.emoji, PartialEmoji):
+                data['emoji'] = self.emoji.to_dict()
+            else:
+                data['emoji'] = {'name': str(self.emoji)}  # type: ignore
+
+                if hasattr(self.emoji, 'id'):
+                    data['emoji']['id'] = int(self.emoji.id)  # type: ignore
+
+        return data
+
+    async def voters(
+        self, *, limit: Optional[int] = None, after: Optional[Snowflake] = None
+    ) -> AsyncIterator[Union[User, Member]]:
+        """Returns an :term:`asynchronous iterator` representing the users that have voted to this answer.
+
+        The ``after`` parameter must represent a user
+        and meet the :class:`abc.Snowflake` abc.
+
+        .. note::
+
+            This can only be called when the parent poll has an attached message.
+
+        Examples
+        --------
+
+        Usage ::
+
+            async for voter in poll_answer.voters():
+                print(f'{voter} has voted for {poll_answer}!')
+
+        Flattening into a list: ::
+
+            voters = [voter async for voter in poll_answer.voters()]
+            # voters is now a list of User
+
+        Parameters
+        ----------
+        limit: Optional[:class:`int`]
+            The maximum number of results to return.
+            If not provided, returns all the users who
+            voted to this poll answer.
+        after: Optional[:class:`abc.Snowflake`]
+            For pagination, voters are sorted by member.
+
+        Raises
+        ------
+        HTTPException
+            Retrieving the users failed.
+
+        Yields
+        ------
+        Union[:class:`User`, :class:`Member`]
+            The member (if retrievable) or the user that has voted
+            to this poll answer. The case where it can be a :class:`Member`
+            is in a guild message context. Sometimes it can be a :class:`User`
+            if the member has left the guild.
+        """
+
+        if not self._message or not self._state:  # Make type checker happy
+            raise RuntimeError('You cannot fetch users in a non-message-attached poll')
+
+        if limit is None:
+            if not self._message.poll:
+                limit = 100
+            else:
+                limit = self.vote_count or 100
+
+        while limit > 0:
+            retrieve = min(limit, 100)
+
+            message = self._message
+            guild = self._message.guild
+            state = self._state
+            after_id = after.id if after else None
+
+            data = await state.http.get_poll_answer_voters(
+                message.channel.id, message.id, self.id, after=after_id, limit=retrieve
+            )
+            users = data['users']
+
+            if len(users) == 0:
+                # No more voters to fetch, terminate process
+                break
+
+            limit -= len(users)
+            after = Object(id=int((users[-1])['id']))
```

Comment:
```suggestion
            after = Object(id=int(users[-1]['id']))
```
stylistic choice, but I think these parens do nothing but make it harder to read.

Reply:
Yeah that was there from previous changes that I reverted and forgot to remove the parentheses. Should be removed by now.